contents/blog/the-posthog-array-1-38-0.mdx

```diff
@@ -0,0 +1,164 @@
+---
+date: 2022-07-01
+title: 'Array 1.38.0: <Joe to add headline here>'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.38.0 introduces <Joe to add detail here>
+---
+
+
+import SessionAnalysis from '../images/blog/array/1-38-0-session-analysis.png'
+import BreakdownBins from '../images/blog/array/1-38-0-breakdown.png'
+
+Want to know more about what we're up to? [Subscribe to HogMail, our new newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [Upgrading PostHog guide](/docs/self-host/configure/upgrading-posthog).
+
+## PostHog 1.38.0 release notes
+
+[ANY IMPORTANT DEPRECATION NOTICES CAN GO HERE]
+
+**Release highlights:**
+
+- New: [Highlight 1]
+- New: [Highlight 2]
+- New: [Highlight 3]
+- Improved: [Highlight 4]
+- Improved: [Highlight 5]
+
+### New: [Highlight 1]
```

Comment:
```suggestion
### New: [Async migration 0005](https://github.com/PostHog/posthog/blob/master/posthog/async_migrations/migrations/0005_person_replacing_by_version.py)

There is a new async migration available, that changes the persons table in ClickHouse to use `version` column instead of `_timestamp` for collapsing entires. This helps avoid various data integrity issues due to race conditions and batching of Kafka messages within plugin-server.

More info about async migrations: https://posthog.com/docs/self-host/configure/async-migrations/overview
More info about async migration 0005: https://github.com/PostHog/posthog/blob/master/posthog/async_migrations/migrations/0005_person_replacing_by_version.py
```